common.gypi

```diff
@@ -216,8 +216,8 @@
             ],
           },],
           ['OS == "android"', {
-            'cflags': [ '-fPIE' ],
-            'ldflags': [ '-fPIE', '-pie' ]
+            'cflags': [ '-fPIC' ],
+            'ldflags': [ '-fPIC' ]
```

Comment:
You only build the shared library, don't you? Does this also work when building the `node` executable? `-fPIE` means Position Independent Executable.

Reply:
It will work, because you can build an executable with PIC code, but it will not be as efficient as PIE code. Not sure about the `-pie` link flag though.

Reply:
Can you check that it still produces a working binary? I have no way to test myself.